c2corg_ui/externs/appx.js

```diff
@@ -84,3 +84,186 @@ appx.BootstrapSliderValues;
  * }}
  */
 appx.auth.RequestChangePassword;
+
+
+/**
+ * @typedef {{
+ *   routes : Array<Object>,
+ *   waypoints : Array<Object>,
+ *   images : Array<Object>,
+ *   users : Array<Object>,
+ *   recent_outings : Array<Object>,
+ * }}
+ */
+appx.DocumentAssociations;
```

Comment:
``all_routes``, ``waypoint_children`` and ``waypoint_parents`` are missing.
Since we do not get all those association types for all documents, ``?`` tags are probably necessary for some of them?